Makefile

```diff
@@ -3,39 +3,39 @@ last_ver = 1.1.6
 long_ver = $(shell git describe --long 2>/dev/null || echo $(short_ver)-0-unknown-g`git describe --always`)
 generated = \
 	build/aiven_extras.control \
-	build/sql/aiven_extras--$(short_ver).sql \
-	build/sql/aiven_extras--$(last_ver)--$(short_ver).sql
-
+	sql/aiven_extras--$(short_ver).sql 
 # for downstream packager
 RPM_MINOR_VERSION_SUFFIX ?=
 
-rpm: rpm-9.6 rpm-10 rpm-11 rpm-12 rpm-13 rpm-14
+# Extension packaging
+EXTENSION = aiven_extras
+MODULE_big = aiven_extras
+OBJS = src/standby_slots.o
+PG_CONFIG ?= pg_config
+DATA = $(wildcard sql/*--*.sql)
+PGXS := $(shell $(PG_CONFIG) --pgxs)
+include $(PGXS)
 
-clean:
-	rm -rf build/ rpm/ aiven-extras-rpm-src.tar
+rpm: rpm-9.6 rpm-10 rpm-11 rpm-12 rpm-13 rpm-14
 
-build/aiven_extras.control: aiven_extras.control.in
+aiven_extras.control: aiven_extras.control.in
```

Comment:
should be `build/aiven_extras.control`

Reply:
Actually, it's the other way around. I removed any mention of build/ directory. 
This is because the PGXS makefile looks for an aiven_extras.control target.